cmake/projects/x264/schemes/url_sha1_x264.cmake.in

```diff
@@ -0,0 +1,342 @@
+# Copyright (c) 2017 Alexandre Pretyman
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+project(Hunter)
+
+include(ExternalProject) # ExternalProject_Add
+
+# Scheme for building x264 libraries only (no programs)
+
+list(APPEND CMAKE_MODULE_PATH "@HUNTER_SELF@/cmake/modules")
+
+include(hunter_fatal_error)
+include(hunter_get_build_flags)
+include(hunter_get_toolchain_binaries)
+include(hunter_report_broken_package)
+include(hunter_status_debug)
+include(hunter_status_print)
+include(hunter_test_string_not_empty)
+
+hunter_status_debug("Scheme: url_sha1_x264")
+
+# Check preconditions
+hunter_test_string_not_empty("@HUNTER_SELF@")
+hunter_test_string_not_empty("@HUNTER_EP_NAME@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_URL@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_SHA1@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_DOWNLOAD_DIR@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_SOURCE_DIR@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_INSTALL_PREFIX@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_CONFIGURATION_TYPES@")
+hunter_test_string_not_empty("@HUNTER_INSTALL_PREFIX@")
+hunter_test_string_not_empty("@HUNTER_GLOBAL_SCRIPT_DIR@")
+
+if("@MSVC@")
+  hunter_user_error("x264 building under MSVC needs to be implemented")
+endif()
+
+# Ideally the running of commands below should be changed in the future to,
+# instead of using the shell variables, control the environment through CMake's
+# ENV{var} and run the commands through execute_process(...)
+
+set(default_path "/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin")
+set(shell_env_path "PATH=@HUNTER_PACKAGE_INSTALL_PREFIX@/bin:${default_path}")
+
+set(d1 "${root_id}/lib/pkgconfig")
```

Comment:
`root_id` is not defined. This code is not covered by `x264` or `odb-boost`, I can reproduce error by building `xau`.

Fixed: https://github.com/ruslo/hunter/commit/dc1a977628e84eae7469eb1d2037c61cdf23a2da